Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
+
+RUN apk add --update ca-certificates \
+    && rm -rf /var/cache/apk/*
```

Comment:
What do you want to say? Should I change it to the code used in cert-operator? Why is this better than the other?